src/main/java/com/eternalcode/parcellockers/parcel/ParcelLockerRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.eternalcode.parcellockers.parcel;
+
+import com.eternalcode.parcellockers.database.JdbcConnectionProvider;
+import com.eternalcode.parcellockers.util.LocationUtil;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public class ParcelLockerRepositoryJdbcImpl implements ParcelLockerRepository {
+
+    private final JdbcConnectionProvider provider;
+
+    public ParcelLockerRepositoryJdbcImpl(JdbcConnectionProvider provider) {
+        this.provider = provider;
+    }
+
+    @Override
+    public CompletableFuture<Void> save(ParcelLocker parcelLocker) {
+        return CompletableFuture.runAsync(() -> this.provider.executeUpdate("INSERT INTO `parcelLockers` (`uuid`, `description`, `location`, `size`) VALUES (" + parcelLocker.getUuid().toString() + ", " + parcelLocker.getDescription() + ", " + parcelLocker.getLocation().toString() + ")"));
```

Comment:
Możesz tutaj zrobić sobie gotowego Stringa ale z tymi `%s` `%d` np. i później używasz metody `String.format()` <- możliwe, że to metoda z innej klasy.